src/frontend/src/components/FilterPanel/style.ts

```diff
@@ -1,12 +1,23 @@
 import styled from "@emotion/styled";
-import { Chip, getColors, getSpacings, InputDropdown } from "czifui";
+import { Chip, getColors, getSpacings, InputDropdown, Props } from "czifui";
```

Comment:
I was taken aback for a moment by czifui exporting something just called `Props`, but I've been trying to work through it for a bit... Am I right in thinking that pretty much every single Props in Aspen has to have that `theme` from the czifui `Props` inside of it, because of how the theme provider works? And so it's reasonable for any/every Aspen component to extend off that czifui `Props`?

I'm really just talking out loud here, but please tell me if I'm wrong or right. I'm trying to get my head around when I should be extending off that czifui `Props` or just creating a new Props just for a component I'm working on and not worrying about extending.

Reply:
TL;DR: Essentially, yes. Most components we style already have the correct set of types because of type inference on the part of TS. It is only when you need to pass a custom prop to a component you plan to style that you actually need to define the type which extends the basic `Prop` set. 

----

For example, if I try to omit this type definition for `StyledFilterPanel` altogether, I end up passing `isOpen` to a styled div. `isOpen` is not a known valid prop for a div, so I need to tell TS that I _meant_ to do that (so that I can use it for styling purposes), and then not forward the prop on to the div that gets rendered in the page. However, if I only define the type as having one prop `isOpen`, then TS complains because the props passed into the string interpolation in emotion's `styled` don't match what emotion provides (something of type `Props`, by default).